src/redist/targets/Crossgen.targets

```diff
@@ -1,7 +1,7 @@
 <Project>
-  <!-- Crossgen is currently not supported on the s390x, ppc64le architecture as using mono instead of CoreCLR. -->
+  <!-- Crossgen is currently not supported on the s390x, ppc64le, loongarch64 architecture as using mono instead of CoreCLR. -->
```

Comment:
@shushanhf, does this comment looks correct to you? At least the second part of this comment is not applicable to LA64. In fact, currently LA64 is not supported on mono, it uses coreclr but still does not ([fully?](https://github.com/dotnet/runtime/pull/85038)) support crossgen2.

Reply:
> @shushanhf, does this comment looks correct to you? At least the second part of this comment is not applicable to LA64. In fact, currently LA64 is not supported on mono, it uses coreclr but still does not ([fully?](https://github.com/dotnet/runtime/pull/85038)) support crossgen2.

Yes, you are right.
Now the crossgen2 of the LA64 is still debugging.
Thanks

Reply:
I am not sure whether the s390x supported mono or not. To avoid making a new mistake, I put another line to describe the loongarch64 only.